packages/snjs/lib/services/settings_service/SNSettingsService.ts

```diff
@@ -0,0 +1,69 @@
+import { SettingPayload } from '@standardnotes/auth';
+import { PureService } from '../pure_service';
+
+import { SNApiService } from '../api/api_service';
+import { SNStorageService } from '../storage_service';
+import { SettingsProvider } from './SettingsProvider';
+import { SettingsCache } from './SettingsCache';
+import { SettingsRepo } from './SettingsRepo';
+import { StorageKey } from '@Lib/storage_keys';
+import { User } from '../api/responses';
+
+export class SNSettingsService extends PureService implements SettingsProvider {
+  private _user?: User;
+  private _provider?: SettingsProvider;
+
+  constructor(
+    private readonly storageService: SNStorageService,
+    private readonly apiService: SNApiService
+  ) {
+    super();
+  }
+
+  private get user(): User {
+    return this.user!;
+  }
+
+  private get provider(): SettingsProvider {
+    return this._provider!;
+  }
+
+  public async initializeFromDisk() {
+    this._user = await this.storageService!.getValue(StorageKey.User);
```

Comment:
Don't think we should be duplicating this from session_manager. Should request it from that service instead.

Reply:
Fixed.